core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -93,13 +107,80 @@
       "<code>MetricReporter</code> interface allows plugging in classes that will be notified " +
       "of new metric creation. The JmxReporter is always included to register JMX statistics.";
   protected static final String METRICS_REPORTER_CLASSES_DEFAULT = "";
+  public static final String SSL_KEYSTORE_LOCATION_CONFIG = "ssl.keystore.location";
```

Comment:
Thanks, @ijuma! @ewencp, I suggest we move forward with this patch as is and we can introduce a helper to include security features later.